migration/v0.26.x-v0.27.x/hns-configuration-to-helm.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+"${here}/../../bin/ck8s" ops kubectl wc annotate hncconfigurations.hnc.x-k8s.io config meta.helm.sh/release-name=hnc
+"${here}/../../bin/ck8s" ops kubectl wc annotate hncconfigurations.hnc.x-k8s.io config meta.helm.sh/release-namespace=hnc-system
+"${here}/../../bin/ck8s" ops kubectl wc label hncconfigurations.hnc.x-k8s.io config app.kubernetes.io/managed-by=Helm
```

Comment:
How come this wasn't added automatically? Because you added this to the chart so helm should be the one deploying this and thus adding these things?

Reply:
The object already exist. It it created by hnc, but we from here on out want to manage the condig object through helm.